docs/_static/css/custom.css

```diff
@@ -52,3 +52,8 @@ div.sponsor-div-box {
    font-size: 0.9rem;
    text-align: center;
  }
+
+ /* Fix version switcher button margin bottom */
+ button.version-switcher__button{
+   margin-bottom: 0em;
+ }
```

Comment:
The version switcher is misaligned on mobile displays due to `margin-bottom` being set as `1em`. This CSS block fixes it.

**Before**
![image](https://user-images.githubusercontent.com/50991099/203155113-f09fa911-b3c9-4153-b046-23a696fd7df3.png)


**After**
![image](https://user-images.githubusercontent.com/50991099/203155070-32668cfb-efac-4394-af96-997b73237362.png)